configs/12.0/packages/libhugetlbfs/sources

```diff
@@ -19,8 +19,8 @@
 # ==========================================
 #
 ATSRC_PACKAGE_NAME="Libhugetlbfs"
-ATSRC_PACKAGE_VER=2.21
-ATSRC_PACKAGE_REV=02df38e93e25
+ATSRC_PACKAGE_VER=2.20
+ATSRC_PACKAGE_REV=e44180072b79
```

Comment:
If libhugetlbfs is downgraded, doesn't it require to diverge AT next from AT 12.0?

Reply:
On master, `libhugetlbfs/sources` is already a regular file (it has been converted by commit ID db8fdbc565bb)